exercises/concept/regular-chatbot/regular-chatbot.js

```diff
@@ -0,0 +1,82 @@
+// @ts-check
+//
+// ☝🏽 The line above enables type checking for this file. Various IDEs interpret
+// the @ts-check directive. It will give you helpful autocompletion on the web
+// and supported IDEs when implementing this exercise. You don't need to
+// understand types, JSDoc, or TypeScript in order to complete this JavaScript
+// exercise, and can completely ignore this comment block and directive.
+
+// 👋🏽 Hi there!
+//
+// On the JavaScript track we provide you with stubs. These stubs provide a
+// starting point to solving the exercise.
+//
+// In general, each variable/constant and each declared function will have a
+// JSDoc comment block above it, explaining what the variable/constant holds or
+// the function is supposed to accomplish.
+//
+// 💡 Often, the JSDoc comment blocks have annotations, such as @param and
+// @returns which are usually highlighted with a different color if the IDE
+// you're in recognizes them. It's these annotations that are used when
+// referring to the constant, variable, or function from somewhere else that
+// IDEs display.
+//
+// You don't need to write these yourself; it is not expected in idiomatic
+// JavaScript, but some companies and style-guides do enforce them.
+//
+// 💡 You're allowed to completely clear a stub before you get started. Often
+// we recommend using the stub, because they are already set-up correctly to
+// work with the tests, which you can find in ./regular-chatbot.spec.js
+//
+// Good luck helping the chatbot!
```

Comment:
Line 2-31 can be removed, we only use this in very early exercises.

Reply:
I didn't know that, thanks for pointing it out 🙏